stream-compositions/cursors/transaction-cursor/pom.xml

```diff
@@ -175,6 +175,25 @@
         </executions>
       </plugin>
 
+            <plugin>
```

Comment:
Wouldn't it be best to centralize this on the parent?

Reply:
I think no, as you do not know which clild uses what generated sources folder(s)

Reply:
@uesleilima may I mark this resolved?